.github/workflows/CI.yml

```diff
@@ -23,10 +23,14 @@ jobs:
         cmake_build_type: ['Debug', 'Release']
     runs-on: ubuntu-20.04
     container: ghcr.io/ornl-mdf/ci-containers/ubuntu:latest
+    continue-on-error: true
```

Comment:
I usually only do this in very specific instances

Reply:
This is intended to work together with the `if: steps.build.outcome == 'success'` parts on the individual steps. From my understanding, if the code gets checked out and built, then following steps can run and fail/pass independent of each other. If any step fails, the job still fails (when `continue-on-error` is given at the job-level).

Reply:
As noted in other comment, I just removed this.